libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -127,12 +127,20 @@ string IRGenerator::generate(Block const& _block)
 	return generator.code();
 }
 
+string IRGenerator::generate(VariableDeclaration const& _varDecl)
+{
+	IRGeneratorForStatements generator(m_context, m_utils);
+	generator.initializeLocalVar(_varDecl);
```

Comment:
Please assert that it is a local variable.

Reply:
It is asserted inside the function, should it be asserted here too?

Reply:
It's fine!